cmd/adapter/adapter_test.go

```diff
@@ -146,3 +147,41 @@ func TestMakePrometheusCAClient(t *testing.T) {
 		}
 	}
 }
+
+func TestParseHeaderArgs(t *testing.T) {
+
```

Comment:
```suggestion
```

Reply:
I've had this in there to be consistent with other functions here. Fixed

Reply:
Ah, I wasn't aware of that. Thanks for fixing it nevertheless :)